app/code/core/Mage/Adminhtml/controllers/Catalog/Product/Action/AttributeController.php

```diff
@@ -188,9 +188,11 @@ public function saveAction()
             $this->_getSession()->addError($attributeName . ': ' . $e->getMessage());
         }
         catch (Mage_Core_Exception $e) {
+            Mage::logException($e);
```

Comment:
I believe the convention is that the error is either a user-error (Mage_Core_Exception) and therefore displayed to the user, or an internal/unexpected error and therefore replaced with a generic error message and logged. So both logging and displaying the same error is not correct. I approve the `logException` in the next block but do not believe this one is necessary.

Reply:
I took your advice and removed this line.